modules/oadp-monitoring.adoc

```diff
@@ -0,0 +1,378 @@
+spec:
+  endpoints:
+  - interval: 30s
+    path: /metrics
+    targetPort: 8085
+    scheme: http
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "velero"
+----
++
+. Apply the `3_create_oadp_service_monitor.yaml`.
++
+[source,terminal]
+----
+$ oc apply -f 3_create_oadp_service_monitor.yaml
+servicemonitor.monitoring.coreos.com/oadp-service-monitor created
+----
++
+. Confirm that the new ServiceMonitor is `Up`.
+In the *Administrator* perspective of the OpenShift Container Platform web console use *Observe/Targets* to view the Metrics Targets. Ensure the `Filter` is unselected or that the `User` source is selected, and type `openshift-adp` in the `Text` search field. Ensure the status for the ServiceMonitor is `Up`.
++
+.OADP metrics targets
+
+image::oadp-metrics-targets.png[OADP metrics targets]
+
+[id="Sample-alerting-rules_{context}"]
+==  Sample alerting rules
+
+The OpenShift Container Platform monitoring stack allows to receive Alerts configured using Alerting Rules. To create an Alerting rule for the OADP project, use one of the Metrics which are scraped with the user workload monitoring.
+
+Refer to xref:../../monitoring/managing-alerts.adoc#managing-alerts[managing alerts] in the OpenShift documentation for detailed instructions on how to create and manage OpenShift Alerts.
+
+.Procedure
+
+. Create a PrometheusRule YAL file with the sample `OADPBackupFailing` alert and save it under `4_create_oadp_alert_rule.yaml` name.
+. In this example an alert displays when there is an increase of failing backups, which indicates new failures, during the 2 last hours that is greater than 0 and this state persisted for at least 5 minutes. If the time of the first increase is less than 5 minutes, the Alert will be in a `Pending` state, after which it will turn into a `Firing` state.
++
+[source,yaml]
++
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: sample-oadp-alert
+  namespace: openshift-adp
+spec:
+  groups:
+  - name: sample-oadp-backup-alert
+    rules:
+    - alert: OADPBackupFailing
+      annotations:
+        description: 'OADP had {{$value | humanize}} backup failures over the last 2 hours.'
+        summary: OADP has issues creating backups
+      expr: |
+        increase(velero_backup_failure_total{job="openshift-adp-velero-metrics-svc"}[2h]) > 0
+      for: 5m
+      labels:
+        severity: warning
+----
++
+. Apply the `4_create_oadp_alert_rule.yaml`, this creates the PrometheusRule in the `openshift-adp namespace`.
++
+[source,terminal]
+----
+$ oc apply -f 4_create_oadp_alert_rule.yaml
+prometheusrule.monitoring.coreos.com/sample-oadp-alert created
+----
++
+. After the Alert is triggered, it can be viewed in the following ways:
+.. In the *Administrator* view under *Observe/Alerting* menu. You must select `User` in the `Filter` box, otherwise by default only the `Platform` alerts are displayed.
+.. In the *Developer* view, select the *Observe* menu.
```

Comment:
```suggestion
.. In the *Administrator* view under *Observe* -> *Alerting* menu, select *User* in the *Filter* box. Otherwise, by default only the *Platform* alerts are displayed.
.. In the *Developer* view, select the *Observe* menu.
```